src/docs/reference/modules/Queries/README.md

```diff
@@ -153,7 +153,7 @@ Here is an example for creating a SQL query from a Queries recipe step:
     "Source": "Sql",
     "Name": "ContentItems",
     "Template": "select * from ContentItemIndex", // json encoded query template
-    "ReturnDocuments": false
+    "ReturnContentItems": false
```

Comment:
This rename is not entirely adequate; Queries can return non-content item documents as well, like Users:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/527b7092-c516-4b40-960e-882e13c9cb8b)

Reply:
Yes, it is. Only content items are returned when the checkbox is selected and every other query type has a `ReturnContentItems` property. This change aligns the property names with the other query types.

Reply:
Users are not content items, but they can still be returned.

Reply:
This is related to this checkbox:

<img width="676" alt="image" src="https://github.com/OrchardCMS/OrchardCore/assets/7062785/a63d81d6-6425-4996-ad92-f623ed481801">

And the change aligns it with this:

https://github.com/OrchardCMS/OrchardCore/blob/80ef1c7a0860a35b0260e833ee6b0161cd53e442/src/OrchardCore.Modules/OrchardCore.Search.Lucene/Services/LuceneQuery.cs#L14

and this:

https://github.com/OrchardCMS/OrchardCore/blob/80ef1c7a0860a35b0260e833ee6b0161cd53e442/src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Models/ElasticQuery.cs#L14

Reply:
OK, I see that e.g. in https://github.com/OrchardCMS/OrchardCore/blob/ma/fix-query-ui-using-document/src/OrchardCore.Modules/OrchardCore.Queries/Sql/SqlQuerySource.cs#L117 whatever the document ID refers to, the result will be cast to a `ContentItem`. So, the name is correct.